app/models/fixed_asset.rb

```diff
@@ -193,6 +193,11 @@ class FixedAsset < Ekylibre::Record::Base
     depreciate! if @auto_depreciate
   end
 
+  def any_financial?
```

Comment:
Could we find a more explicit method name? Something like 'on_closed_periods?' maybe.